quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/metadata.json

```
@@ -1,10 +1,10 @@
 {
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "type": "QuickStart",
-  "itemDisplayName": "SQL Server SQL Data Warehouse with Transparent Encryption.",
-  "description": "Creates a SQL Server and a SQL Data Warehouse with Transparent Data Encryption.",
-  "summary": "This template creates a SQL Server and a SQL Data Warehouse with Transparent Data Encryption.",
+  "itemDisplayName": "Dedicated SQL pool with Transparent Encryption.",
+  "description": "Creates a SQL Server and a Dedicated SQL pool (formerly SQL DW) with Transparent Data Encryption.",
+  "summary": "This template creates a SQL Server and a Dedicated SQL pool (formerly SQL DW) with Transparent Data Encryption.",
   "githubUsername": "cmendible",
   "docOwner": "julieMSFT",
```

```suggestion
  "docOwner": "WilliamDAssafMSFT",
```
Please go ahead and accept this suggestion to also update this codeowner, Julie is no longer with MSFT.

---

Done!